common/test/acceptance/tests/lms/test_certificate_web_view.py

```diff
@@ -1,8 +1,14 @@
 """
 Acceptance tests for the certificate web view feature.
 """
+from common.test.acceptance.tests.helpers import (
+    UniqueCourseTest,
+    EventsTestMixin,
+    load_data_str,
+    get_element_padding,
+    get_modal_alert,
+)
```

Comment:
For import orders: see http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#imports-order